dbflute-engine/src/main/java/org/dbflute/logic/replaceschema/process/arrangebefore/DfArrangeBeforeRepsProcess.java

```diff
@@ -207,7 +210,7 @@ protected void copyFile(File src, File dest) {
             dest.delete();
         }
         try {
-            FileUtils.getFileUtils().copyFile(src, dest);
+            Files.copy(src.toPath(), dest.toPath(), StandardCopyOption.COPY_ATTRIBUTES);
```

Comment:
I will write tests of ArrangeBeforeReps

Reply:
I wrote.

https://github.com/dbflute-test/dbflute-test-active-hangar/blob/master/src/test/java/org/docksidestage/hangar/dbflute/whitebox/world/diffworld/WxDiffworldTest.java